Share/Share.Plugin.Abstractions/ShareUIActivityType.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Plugin.Share.Abstractions
+{
+    public enum ShareUIActivityType
+    {
+        // iOS 6+
+
+        /// <summary>
+        /// The object assigns the image to a contact.
```

Comment:
Descriptions are copied from the iOS reference: https://developer.apple.com/reference/uikit/uiactivitytype